### ISSUE TITLE
Possible error passing additional arguments in summarise

### ISSUE THREAD

**Issue by rabutler-usbr** [tidyverse/dplyr#4163]
In ensuring that [RWDataPlyr](https://github.com/BoulderCodeHub/RWDataPlyr) will work with the upcoming dplyr release, I have come across an issue that I'm not sure is expected or not based on my reading of the breaking changes in v0.8.0. 

I believe the behavior of using `sum(x, na.rm = TRUE)` in a call to `summarise()` has changed between versions v0.7.x and v0.8.0. 

The following illustrates the differences in the two versions, where 1 out of 12 values is `NaN` and the desired behavior is to sum all the other values, skipping the `NaN` value(s). 

**In v0.7.8**

``` r
library(dplyr)
df <- data.frame(
  month = rep(month.abb, 2), 
  year = c(rep(2019, 12), rep(2020, 12)),
  value = c(rep(1, 15), NaN, rep(1, 8))
)

df %>% group_by(year) %>% summarise(value = sum(value, na.rm = TRUE))
#> # A tibble: 2 x 2
#>    year value
#>   <dbl> <dbl>
#> 1  2019    12
#> 2  2020    11
```

<sup>Created on 2019-02-05 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**In v0.8.0**

``` r
library(dplyr)
df <- data.frame(
    month = rep(month.abb, 2), 
    year = c(rep(2019, 12), rep(2020, 12)),
    value = c(rep(1, 15), NaN, rep(1, 8))
)
    
df %>% group_by(year) %>% summarise(value = sum(value, na.rm = TRUE))
#> # A tibble: 2 x 2
#>    year value
#>   <dbl> <dbl>
#> 1  2019    12
#> 2  2020   NaN
```

<sup>Created on 2019-02-05 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

The same issue also appears if using `na.rm = TRUE` in `min()`, or `max()`.

If this is intended, I'll happily update my RWDataPlyr tests accordingly, but I couldn't see where this was intended. 

Thanks!

**Comment by romainfrancois** [comment]
Looks related to this PR #4109. 

``` r
library(dplyr, warn.conflicts = FALSE)
df <- data.frame(
  month = rep(month.abb, 2), 
  year = c(rep(2019, 12), rep(2020, 12)),
  value = c(rep(1, 15), NaN, rep(1, 8))
)

df %>% 
  group_by(year) %>% 
  summarise(
    hybrid = sum(value, na.rm = TRUE), 
    standard = +sum(value, na.rm = TRUE)
  )
#> # A tibble: 2 x 3
#>    year hybrid standard
#>   <dbl>  <dbl>    <dbl>
#> 1  2019     12       12
#> 2  2020    NaN       11
```

on it.

**Comment by rabutler-usbr** [comment]
Thanks for the quick response @romainfrancois. 

So does this mean that I should be using the `+sum()` to ensure that NaN's are removed?

**Comment by romainfrancois** [comment]
No, `+sum()` is just so that hybrid evaluation was not triggered. With the fix in #4164, you should be able to use `sum()` as usual.
